### PR TITLE
chore: Add README and getting-started material

### PR DESCRIPTION
**Pull request by sylvain-reynaud**

### CODE REVIEW

**Review by mflagey — APPROVED** [review]

**Review by mflagey — COMMENTED** [review]
Please fix CI

File: .docker/docker-compose.dev.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+services:
+  firepilot-build:
+    build:
+      context: "${PWD}/.docker"
+      dockerfile: "Dev.Dockerfile"
+    container_name: firepilot-build
+    volumes:
+      - ${PWD}:/build
+    command: "cargo build"
```
Comment (AlexandreBrg) [inline .docker/docker-compose.dev.yml:11]: Hello, why would we want to have a docker compose ? It's only a library

Also, maybe you were thinking on having an homogeneous way to compile the project, and IMO it's handled by the CI. Moreover, building through docker have huge performance issues (going from few seconds of build in bare metal to few minutes in docker)

**Review by MaloPolese — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+rootfs.ext4 example : https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/rootfs/bionic.rootfs.ext4
+
+vmlinux.bin example : https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/kernels/vmlinux.bin
+
+```rust
+use firepilot::microvm::{BootSource, Config, Drive, MicroVM};
+use firepilot::Firecracker;
+
+let firecracker = Firecracker::new(Some(firepilot::FirecrackerOptions {
+                    command: Some(PathBuf::from("/app/firecracker")),
```
Comment (mflagey) [inline README.md:33]: /app is not very clear in this context

File: README.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+                    ..Default::default()
+                }))
+                .unwrap();
+
+let vm = MicroVM::from(Config {
+    boot_source: BootSource {
+        kernel_image_path: PathBuf::from("/app/vmlinux.bin"),
```
Comment (mflagey) [inline README.md:40]: same here

File: README.md [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+        boot_args: None,
+        initrd_path: None,
+    },
+    drives: vec![Drive {
+        drive_id: "rootfs".to_string(),
+        path_on_host: PathBuf::from("/app/rootfs.ext4"),
```
Comment (mflagey) [inline README.md:46]: same here

**Pull request merged**